src/main/kotlin/no/nav/syfo/aktivitetskrav/api/AktivitetskravApi.kt

```diff
@@ -100,9 +100,11 @@ fun Route.registerAktivitetskravApi(
                     throw IllegalArgumentException("Failed to vurdere aktivitetskrav: personIdent on aktivitetskrav differs from request")
                 }
 
-                val aktivitetskravVurdering = requestDTO.toAktivitetskravVurdering(
-                    createdByIdent = call.getNAVIdent(),
-                )
+                val aktivitetskravVurdering = try {
+                    requestDTO.toAktivitetskravVurdering(createdByIdent = call.getNAVIdent())
+                } catch (e: Exception) {
+                    throw IllegalArgumentException("Failed to vurdere aktivitetskrav: ${e.message}")
+                }
```

Comment:
For at `!!` i lagingen av domene objekter ikke skal returnere 500, men heller 400

Reply:
Kunne man brukt `requireNotNull` i stedet for !! i oppretelsen av domeneobjekter i stedet?

Reply:
Aha! Ja det ser jo ut som det resulterer i det samme, bare uten `try catch`'en 🙌 Endrer